website/content/docs/job-specification/resources.mdx

```diff
@@ -86,5 +86,49 @@ resources {
   }
 }
 ```
+## Memory Oversubscription
+
+Setting task memory limits is a challenging act of balance. Job authors must set
+a memory limit for each task — if it’s too low, the task may be interrupted if
+it exceeds the limit unexpectedly; if it’s too high, the cluster is left
+underutilized and resources are wasted. Job authors usually set limits based on
+their task's typical memory usage and then add a safety margin to handle
+unexpected load spikes or uncommon scenarios. That means that in some clusters,
+20-40% of the cluster memory may be reserved but unused.
+
+To help minimize reserved but not used cluster memory, Nomad 1.1 lets job
+authors set two separate memory limits:
```

Comment:
Because we took this from the blog post it's reading as a bit "marketing-y" and it includes operator behaviors we're looking to mitigate, rather than focusing on . I'd tighten this up as follows:

```suggestion
Setting task memory limits requires balancing the risk of interrupting tasks
against the risk of wasting resources. If a task memory limit is set too low,
the task may exceed the limit and be interrupted; if the task memory is too
high, the cluster is left underutilized.

To help maximize cluster memory utilization while allowing a safety margin for
unexpected load spikes, Nomad 1.1. lets job authors set two separate memory
limits:
```